cmd/performance-profile-creator/README.md

```diff
@@ -0,0 +1,55 @@
+# Performance Profile Creator (PPC)
+
+A tool to automate the process of creating Performance Profile using the user supplied profile parameters.
+
+## Software Components
+
+1. A CLI tool part of the PAO image
+1. A wrapper script <!--(TODO: Add link to where the wrapper script is placed) --> automates the access to PPC tool. It pulls the PAO image, supplies arguments and runs the tool inside Performance Addon Operator container with performance-profile-creator entrypoint.
+
+## Flow
+1. The user supplies the command line args to the tool by using a wrapper script.
+1. The wrapper triggers the must-gather tool.
+1. A must-gather output file is passed to PPC as an argument.
+1. The wrapper decompresses the must-gather output file and “mount” the ghw snapshot in order to inspect the hardware, and passes the snapshot to PPC. The ghw snapshots can also be supplied by the user may directly.
```

Comment:
this is an implementation detail, so depending on the target audience (users vs developers) could be deferred to a later section or to a different document entirely.

From user perspective the most relevant content seems:
1. PPC consumes a must-gather output
2. PPC may run must-gather directly if the must-gather output is not given
3. output is a bunc of YAML data (PAO profile + NTO tuned part)

Reply:
Perfect, will update as above. Thanks for the suggestion